### PR TITLE
Add QBittorrent's alternative speed switch doc

### PR DESCRIPTION
**Pull request by Sebclem**
## Proposed change

Add QBittorrent's alternative speed switch doc added by this PR: https://github.com/home-assistant/core/pull/107637

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/107637
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Introduced control for the `Alternative Speed` switch in the qBittorrent integration to manage alternative speed settings within Home Assistant.
  - Enhanced documentation with a dedicated section for the Alternative Speed switch, improving user interaction and understanding of its functionality.

### CODE REVIEW

**Comment by frenck** [comment]
Hi there @Sebclem 👋 

This PR seems to have collected a merge conflict. Could you take a look and resolve it?
Afterward, please mark the PR ready for review, so it can be picked up.

Thanks 👍 

../Frenck

File: source/_integrations/qbittorrent.markdown [hunk 1]
```diff
@@ -35,6 +37,10 @@ The qBittorrent integration will add the following sensors:
 - `sensor.qbittorrent_inactive_torrents`: The current inactive torrents in qBittorrent.
 - `sensor.qbittorrent_paused_torrents`: The current paused torrents in qBittorrent.
 
+## Switch
+
+`Alternative Speed` is added, controlling qBittorrent's alternative speed
```
[the remark this answers is not in the document]
Comment (Sebclem) [reply source/_integrations/qbittorrent.markdown]: make me a draft

[the remark this answers is not in the document]
Comment (Sebclem) [reply source/_integrations/qbittorrent.markdown]: @frenck (sorry I don't know who else to ping) did we really need this kind of description, or can I skip this ?
I have changed to this: 
```markdown
## Switch

The qBittorrent integration will add the following switch:

- `Alternative Speed`: Allow you to enable and disable qBittorrent's alternative speed
```

File: source/_integrations/qbittorrent.markdown [hunk 2]
```diff
@@ -12,10 +12,12 @@ ha_codeowners:
   - '@finder39'
 ha_platforms:
   - sensor
+  - switch
 ha_integration_type: service
 ---
 
 The `qbittorrent` platform allows you to monitor your downloads with [qBittorrent](https://www.qbittorrent.org/) from within Home Assistant and setup automations based on the information.
+You can control the alternative speed via the `Alternative Speed` switch.
```
Comment (c0ffeeca7) [inline source/_integrations/qbittorrent.markdown]:
```suggestion
You can control the alternative speed via the `Alternative speed` switch.
```

File: source/_integrations/qbittorrent.markdown [hunk 3]
```diff
@@ -35,6 +37,12 @@ The qBittorrent integration will add the following sensors:
 - `sensor.qbittorrent_inactive_torrents`: The current inactive torrents in qBittorrent.
 - `sensor.qbittorrent_paused_torrents`: The current paused torrents in qBittorrent.
 
+## Switch
+
+The qBittorrent integration adds the following switch:
+
+- `Alternative Speed`: Allows you to enable and disable qBittorrent's alternative speed.
```
Comment (c0ffeeca7) [inline source/_integrations/qbittorrent.markdown]:
```suggestion
- `Alternative speed`: Allows you to enable and disable qBittorrent's alternative speed.
```

File: source/_integrations/qbittorrent.markdown [hunk 4]
```diff
@@ -35,6 +37,12 @@ The qBittorrent integration will add the following sensors:
 - `sensor.qbittorrent_inactive_torrents`: The current inactive torrents in qBittorrent.
 - `sensor.qbittorrent_paused_torrents`: The current paused torrents in qBittorrent.
 
+## Switch
+
+The qBittorrent integration adds the following switch:
+
+- `Alternative speed`: Allows you to enable and disable qBittorrent's alternative speed.
```
Comment (c0ffeeca7) [inline source/_integrations/qbittorrent.markdown]:
```suggestion
- `Alternative speed`: Allows you to enable or disable qBittorrent's alternative speed.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Sebclem :+1:

**Pull request merged**